Add VMWare Tanzu notes. Also differentiate between supported and tested k8s (#719)
stackabletech/documentation: modules/ROOT/pages/kubernetes/huawei-cloud.adoc

= Huawei cloud

> Huawei cloud uses a non-standard Kubelet state directory.

Reviewer: ```suggestion
Huawei Cloud uses a non-standard Kubelet state directory.
```

> Therefore installing secret-operator and listener-operator on Huawei cloud requires a special treatment.
> Please read on the xref:secret-operator:installation.adoc#_huawei_cloud[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_huawei_cloud[listener-operator installation guide] for details.

Reviewer: ```suggestion
Therefore, installing the secret-operator and listener-operator on Huawei Cloud requires special handling.
Please refer to the xref:secret-operator:installation.adoc#_huawei_cloud[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_huawei_cloud[listener-operator installation guide] for details.
```
